lib/ts/recipe/thirdparty/components/themes/styles.ts

```diff
@@ -40,46 +40,53 @@ export function getStyles(palette: NormalisedPalette): NormalisedDefaultStyles {
 
         providerButton: {
             // Default button height in 34
-            minHeight: "34px",
+            minHeight: "32px",
             // this will allow the button to scale with different font sizes and text lengths
-            height: "auto !important",
+            height: "auto",
             display: "flex",
             flexDirection: "row",
+            alignItems: "center",
             paddingLeft: "0px",
             paddingRight: "0px",
             // This makes the button look somewhat cleaner if the text wraps
             paddingTop: "2px",
             paddingBottom: "2px",
+            "&:hover": {
+                filter: "none !important",
+                backgroundColor: "rgb(250,250,250)",
+            },
         },
 
         providerButtonLeft: {
-            width: "40px",
+            marginRight: "8px",
+            marginLeft: "78px",
         },
 
         providerButtonLogo: {
             height: "30px",
             display: "flex",
-            borderRight: "1px solid rgba(255, 255, 255, 0.6)",
         },
 
         providerButtonLogoCenter: {
+            display: "flex",
             margin: "auto",
         },
 
         providerButtonText: {
-            margin: "auto",
+            fontFamily: "Rubik",
+            fontWeight: 400,
             textAlign: "center",
             justifyContent: "center",
         },
 
-        providerGoogle: getButtonStyle(providerColors.google, "white"),
-        providerGitHub: getButtonStyle(providerColors.github, "white", true),
-        providerTwitter: getButtonStyle(providerColors.twitter, "white"),
-        providerFacebook: getButtonStyle(providerColors.facebook, "white"),
-        providerApple: getButtonStyle(providerColors.apple, "white", true),
+        providerGoogle: getButtonStyle(providerColors.google, "black"),
```

Comment:
We can still keep these styles separate as an option for customization.

Reply:
something like 
`const providerTextColors = { /*...*/ }` ?

Reply:
I'd prefer if you added something like `const defaultProviderButtonStyle = getButton...` and added that to props here.